telemetry/definitions/commonDefinitions.json

```diff
@@ -2771,6 +2771,19 @@
                 { "type": "amazonqGenerateApproachLatency" }
             ]
         },
+        {
+            "name": "amazonq_startConversationInvoke",
+            "description": "Captures startConversation process",
```

Comment:
the description doesn't tell me anything useful.

how does this differ from `amazonq_startChat` (which still mentions "/dev", is that correct)?

and the description on `amazonq_endChat` mentions "end of the conversation".

Reply:
This one is specifically related to invoking the api, I will update. It is more evident in this work in progress PR https://github.com/aws/aws-toolkit-vscode/pull/4157